packages/calcite-components/src/components/input-date-picker/input-date-picker.scss

```diff
@@ -1,55 +1,114 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ * 
+ * @prop --calcite-input-date-picker-border-color: Specifies the component's border color.
+ * @prop --calcite-input-date-picker-shadow: Specifies the component's shadow.
```

Comment:
I'm still not sure about the final token set here. There are some that seem confusing or redundant. For example,
 
`--calcite-input-date-picker-icon-color` + `--calcite-input-date-picker-input-icon-color`
 
Both of these props' targets style an icon that are displayed within the input field.
 
`--calcite-input-date-picker-shadow` + `--calcite-input-date-picker-shadow`
 
Why would we need to style these separately?

I could be missing something, but if this makes sense, can you work with @macandcheese to refine this into a simpler, more foundational token set, if possible?